build.sbt

```diff
@@ -292,6 +292,24 @@ lazy val scioBigtable: Project = Project(
   scioTest % "it"
 ).configs(IntegrationTest)
 
+lazy val scioElsticsearch: Project = Project(
+  "scio-elasticsearch",
+  file("scio-elasticsearch")
+).settings(
+  commonSettings ++ itSettings,
+  description := "Scio add-on for Writing to Elasticsearch Cluster",
+  libraryDependencies ++= Seq(
+    "org.apache.beam" % "beam-sdks-java-io-google-cloud-platform" % beamVersion,
+    "com.google.auto.value" % "auto-value" % "1.2",
+    "joda-time" % "joda-time" % jodaTimeVersion,
+    "org.elasticsearch" % "elasticsearch" % "2.1.0",
```

Comment:
Technically we should be able to use latest client. However this code is tested under version 2.1.0 so not sure about changing it without testing the code against ES cluster. Should we wait and see?